Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -121,7 +121,8 @@ final class PackageToolTests: XCTestCase {
             XCTAssert(textOutput.hasPrefix("Name: SwiftCMixed"))
             XCTAssert(textOutput.contains("    C99name: CExec"))
             XCTAssert(textOutput.contains("    Name: SeaLib"))
-            XCTAssert(textOutput.contains("   Sources: main.swift"))
+            XCTAssert(textOutput.contains("    Sources:"))
```

Comment:
More unit tests to follow.  Since the plain text output never had a guaranteed output format (that's what JSON is for), it should not be a problem to change it here.  It's a lot more readable to have the source files listed on separate lines than in a long comma-separated list.